.ci/assets/openshift/pulp-catalog-source.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: my-pulp-catalog
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: quay.io/pulp/pulp-operator:latest
```

Comment:
Should point to an index, not the operator itself. https://github.com/operator-framework/operator-registry#building-an-index-of-operators-using-opm

Reply:
thanks for the link.